data/xml/2022.inlg.xml

```diff
@@ -454,7 +479,8 @@
       <title>niksss at <fixed-case>H</fixed-case>inglish<fixed-case>E</fixed-case>val: Language-agnostic <fixed-case>BERT</fixed-case>-based Contextual Embeddings with Catboost for Quality Evaluation of the Low-Resource Synthetically Generated Code-Mixed <fixed-case>H</fixed-case>inglish Text</title>
       <author><first>Nikhil</first><last>Singh</last></author>
       <pages>31-34</pages>
-      <abstract>This paper describes the system description for the HinglishEval challenge at INLG 2022. The goal of this task was to investigate the factors influencing the quality of the code- mixed text generation system. The task was divided into two subtasks, quality rating pre- diction and annotators’ disagreement predic- tion of the synthetic Hinglish dataset. We at- tempted to solve these tasks using sentence- level embeddings, which are obtained from mean pooling the contextualized word embed- dings for all input tokens in our text. We experimented with various classifiers on top of the embeddings produced for respective tasks. Our best-performing system ranked 1st on subtask B and 3rd on subtask A. We make our code available here: <url>https://github.com/nikhilbyte/Hinglish-qEval</url></abstract>
+      <abstract>This paper describes the system description for the HinglishEval challenge at INLG 2022. The goal of this task was to investigate the factors influencing the quality of the code- mixed text generation system. The task was divided into two subtasks, quality rating pre- diction and annotators’ disagreement predic- tion of the synthetic Hinglish dataset. We at- tempted to solve these tasks using sentence- level embeddings, which are obtained from mean pooling the contextualized word embed- dings for all input tokens in our text. We experimented with various classifiers on top of the embeddings produced for respective tasks. Our best-performing system ranked 1st on subtask B and 3rd on subtask A. We make our code available here: <url>https://github.com/nikhilbyte/Hinglish-qEval</url>
+      </abstract>
```

Comment:
Weird that this got reformatted

Reply:
Maybe you can amend the commit and do a force push to undo this @mjpost?

Reply:
I just fixed it, along with a handful of hyphenations.

Reply:
Actually, grepping for the pattern, it seems that any time an abstract ends with `</url>`, the abstract closing tag got pushed to the next line. I wonder if this was introduced in #2806.